lib/Sema/ConstraintSystem.cpp

```diff
@@ -2583,6 +2583,18 @@ static void diagnoseOperatorAmbiguity(ConstraintSystem &cs,
   if (!applyExpr)
     return;
 
+  auto isNameOfStandardComparisonOperator = [](StringRef opName) -> bool {
+    return opName == "==" || opName == "!=" || opName == "===" ||
```

Comment:
Can you use Identifier::is() instead of comparing StringRefs here?

Reply:
Sure, I'll change it!